package.json

```diff
@@ -2,18 +2,19 @@
   "name": "@govtechsg/open-attestation",
   "version": "0.0.0-development",
   "description": "## Usage",
-  "main": "dist/cjs/index.js",
-  "module": "dist/esm/index.js",
-  "browser": "dist/index.umd.js",
-  "types": "dist/types/index.d.ts",
+  "type": "module",
+  "exports": {
+    ".": "./dist/index.js"
+  },
+  "types": "dist/index.d.ts",
+  "engines": {
+    "node": ">=12"
```

Comment:
the engine version is different from that specified in `.nvmrc`.

Reply:
Nope engines is for runtimes, nvm is for dev